File: app/src/free/java/br/com/helpdev/validador/FragValidadores.java
package br.com.helpdev.validador;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;

public class FragValidadores extends FragValidadoresAbs {

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        AdView mAdView = (AdView) view.findViewById(R.id.adView);
        AdRequest adRequest = new AdRequest.Builder()
                .addTestDevice(AdRequest.DEVICE_ID_EMULATOR)
                .addTestDevice("ED22D380D881867345E6858189EB4D35")
                .build();
        mAdView.loadAd(adRequest);
    }
}

File: app/src/free/java/br/com/helpdev/validador/FragValidadores.kt
package br.com.helpdev.validador

import android.os.Bundle
import android.view.View
import com.google.android.gms.ads.AdRequest
import com.google.android.gms.ads.AdView

class FragValidadores : FragValidadoresAbs() {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val mAdView = view.findViewById<AdView>(R.id.adView)
        val adRequest = AdRequest.Builder()
                .addTestDevice(AdRequest.DEVICE_ID_EMULATOR)
                .addTestDevice("ED22D380D881867345E6858189EB4D35")
                .build()
        mAdView.loadAd(adRequest)
    }
}
